### ISSUE TITLE
The training.

### ISSUE THREAD

**Issue by zqgao22** [ml4bio/Dense-Homolog-Retrieval#21]
Hi the DHR team,

Thank you for your contributions for open source. Could you please provide a script for training? We would like to follow your work closely and do some training on new datasets. 

Thanks.

**Comment by jxxll** [comment]
Your work is truly impressive! I believe it will be incredibly beneficial for my graduation project. Would you provide a training script? Thank you!

**Comment by heathcliff233** [comment]
Sorry for the late reply. We did not release the training code because it has not been close to publish quality. The loss and dataset implementations are already in this repo. If you would like more training reference, please take a look at 
https://github.com/heathcliff233/pldpr
